editors-processor/src/test/resources/org/gwtproject/editor/processor/common/WidgetImplementsAsEditorLong.java

```diff
@@ -0,0 +1,35 @@
+package org.gwtproject.editor.processor.common;
+
+import org.gwtproject.editor.client.EditorError;
+import org.gwtproject.editor.client.HasEditorErrors;
+import org.gwtproject.editor.client.LeafValueEditor;
+import org.gwtproject.editor.client.TakesValue;
+
+import java.util.List;
+
+public class WidgetImplementsAsEditorLong
```

Comment:
See the testing package, FakeLeafValueEditor would be easier to use for most of these, since you actually aren't testing these other interfaces

Reply:
The test tool requires resources, not Java classes. Not sure if this will work.

Reply:
The org.gwtproject.editor.testing stuff is in the main editors jar, so you can reference it from here as easily as you can reference Editor itself, and cut down on some extra types.

Reply:
Replace editors with FakeLeafEditor.